### PR TITLE
fix: default index could be disabled

### LINKED ISSUE #41: duplicated URLs serving index.html
Hello,

About html files and the fact both `http://app/` and `http://app/index.html` serve the same index.html, is there a way to prevent from serving the second route?

I guess redirecting using a router is an option, but it requires extra router processing on each request,

Thanks!

### PR DESCRIPTION
**Pull request by coderhaoxin**
closes #41

### CODE REVIEW

**Comment by tejasmanohar** [comment]
:+1:

File: test/index.js [hunk 1]
```diff
@@ -90,6 +90,18 @@ describe('serve(root)', function(){
           .expect('html index', done);
         })
       })
+
+      describe('when disabled', function(){
+        it('should use index.html', function(done){
```
Comment (tj) [inline test/index.js]: "should not use"? otherwise LGTM

Comment (coderhaoxin) [inline test/index.js]: sorry :cry:

**Comment by tj** [comment]
woot thanks!

**Comment by danneu** [comment]
looks good and solves the problem. 

though it's weird that the koa ecosystem seemed to standardize on nullifying non-boolean options with `false` instead of `null`.

**Comment by tj** [comment]
`index: null` reads weird IMO

**Comment by coderhaoxin** [comment]
1.5.2 published

**Comment by tejasmanohar** [comment]
>though it's weird that the koa ecosystem seemed to standardize on nullifying non-boolean options with `false` instead of `null`.
+1, weird indeed. not semantic. i generally like to avoid null values :)

**Comment by danneu** [comment]
@tj just seems funny to define behavior for half a boolean.

it makes more sense to use `null` if the distinction between `undefined` and `null` is "not set at all" and "set, but nullified", respectively.

though i just wanted to noncommittally throw some paint onto the bikeshed on my way by.

**Pull request merged**